cves/kernel/CVE-2020-28974.yml

```diff
@@ -106,7 +109,7 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 1da177e4c3f41524e886b7f1b8a0c1fc7321cac2
-  note: Discovered automatically by archeogit.
+  note: Discovered automatically by archeogit. Huge patch with more than 17,000 changed files and more than 6,700,000 additions
 - commit: edab558feba1e2826ae8d65506168d7ccea7aad9
   note: Discovered automatically by archeogit.
 - commit: ac751efa6a0d70f2c9daef5c7e3a92270f5c2dff
```

Comment:
Should review these to check if they had any actual impact on the vulnerability.